cg/constants/demultiplexing.py

```diff
@@ -19,11 +19,15 @@ class FlowCellType(StrEnum):
     HISEQ: str = "hiseq"
 
 
+class SampleSheetHeader(StrEnum):
+    FLOW_CELL_ID: str = "FCID"
```

Comment:
```suggestion
    FLOW_CELL_COLUMN_NAME: str = "FCID"
```
the constant does not really refer to a flow cell id (i.e. HCC7151AD ... ) maybe being more explicit that it is a column name that is being looked for is more appropriate.